docs/UserGuide.adoc

```diff
@@ -11,226 +12,341 @@ ifdef::env-github[]
+* The INDEX refers to the index number shown in the most recent listing.
+* The INDEX *must be a positive integer* 1, 2, 3, ...
 ****
 
 Examples:
 
 * `list` +
-`delete 2` +
-Deletes the 2nd person in the address book.
+`delete 2 pw/ADMIN_PASSWORD` +
+Deletes the 2nd employee in PTMan.
 * `find Betsy` +
-`delete 1` +
-Deletes the 1st person in the results of the `find` command.
+`delete 1 pw/ADMIN_PASSWORD` +
+Deletes the 1st employee in the results of the `find` command.
 
-=== Selecting a person : `select`
+==== Resetting Password for an employee : `rp`
 
-Selects the person identified by the index number used in the last person listing. +
-Format: `select INDEX`
+Reset the password for employee from PTMan. +
+*Format:* `rp INDEX pw/ADMIN_PASSWORD` +
 
 ****
-* Selects the person and loads the Google search page the person at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* `1, 2, 3, ...`
+* The password will reset for employee at the specified INDEX.
+* The INDEX refers to the index number shown in the most recent listing.
+* The INDEX *must be a positive integer* 1, 2, 3, ...
 ****
 
 Examples:
 
 * `list` +
-`select 2` +
-Selects the 2nd person in the address book.
+`rp 2 pw/ADMIN_PASSWORD` +
+Reset password for the 2nd employee in PTMan.
 * `find Betsy` +
-`select 1` +
-Selects the 1st person in the results of the `find` command.
-
-=== Listing entered commands : `history`
-
-Lists all the commands that you have entered in reverse chronological order. +
-Format: `history`
-
-[NOTE]
-====
-Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
-====
+`rp 1 pw/ADMIN_PASSWORD` +
+Reset password for the 1st employee in the results of the `find` command.
 
 // tag::undoredo[]
-=== Undoing previous command : `undo`
+==== Undoing previous command : `undo`
 
-Restores the address book to the state before the previous _undoable_ command was executed. +
-Format: `undo`
+Restores PTMan to the state before the previous _undoable_ command was executed. +
+*Format:* `undo pw/ADMIN_PASSWORD` +
+*Shorthand:* `u pw/ADMIN_PASSWORD`
 
 [NOTE]
 ====
-Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit` and `clear`).
+Undoable commands: Commands that modify PTMan's data (`add`, `delete`, `edit` and `clear`).
 ====
 
 Examples:
 
-* `delete 1` +
+* `delete 1 pw/ADMIN_PASSWORD` +
 `list` +
-`undo` (reverses the `delete 1` command) +
+`undo pw/ADMIN_PASSWORD` (reverses the `delete 1` command) +
 
-* `select 1` +
-`list` +
-`undo` +
+* `list` +
+`undo pw/ADMIN_PASSWORD` +
 The `undo` command fails as there are no undoable commands executed previously.
 
-* `delete 1` +
-`clear` +
-`undo` (reverses the `clear` command) +
-`undo` (reverses the `delete 1` command) +
+* `delete 1 pw/ADMIN_PASSWORD` +
+`clear pw/ADMIN_PASSWORD` +
```

Comment:
I think asking for the password to be entered everytime is a hassle + security issue. You may have someone peeping into your screen! :stuck_out_tongue_winking_eye: